### ISSUE TITLE
github splash page typo

### ISSUE THREAD

**Issue by SiccarPoint** [landlab/landlab#927]
A typo lurks in our splash page (landlab.github.io) quick install examples:

Near the bottom:

```
However, if you are running an Anaconda distribution, we now recommend you uninstall with pip and replace with a conda install. Simply do this:

> pip uninstall landlab

> conda install landlab -c landlab -
```

That final trailing `-` I believe to be a pure typo for removal. Someone correct me if I'm wrong?

**Comment by kbarnhart** [comment]
You are right that it is typo. But we also should be using the conda-forge
channel instead of the landlab channel.

So it should be

conda install landlab -c conda-forge

I’d just make the change and push, but I’m on the bus.

Once we have a new release there will be a pip version too.
